dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -2892,6 +2905,16 @@ public void testKeysetWithoutPageRequest() {
         assertEquals(false, page.hasPrevious());
     }
 
+    /**
+     * Use a repository method that performs a JDQL query using LEFT function
+     * to obtain the beginning of a String value.
+     */
+    @Test
+    public void tesLeftFunction() {
```

Comment:
```suggestion
    public void testLeftFunction() {
```
This can wait for a future PR

Reply:
Good catch. Yes, I'll fix this typo in the next PR because this one has a passing build and it's only a test case method name.